vms/platformvm/upgrade/times.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package upgrade
+
+import "time"
+
+type Times struct {
```

Comment:
Can we call this `Config`? This is technically a network-configurable value so just keeping it in-line with other similar structs.

Reply:
Done